CHANGES.md

```diff
@@ -14,6 +14,7 @@ Release Notes.
 * Change the dateformat of logs to `yyyy-MM-dd HH:mm:ss.SSS`(was `yyyy-MM-dd HH:mm:ss:SSS`).
 * Fix NPE in elasticsearch plugin.
 * Grpc plugin support trace client async generic call(without grpc stubs), support Method type: `UNARY`、`SERVER_STREAMING`.
+* Enhance Apache ShenYu (incubating) plugin: support trace `grpc`,`sofarpc`,`motan`,`tars` rpc proxy.
```

Comment:
tars? I am concerned about this, we don't have tars plugin for the service's RPC. This new feature seems a little misguided.

Also, supported-list.md should be updated too.

Reply:
This can only be supported to the gateway level, not to the service provider, because there is no skywalking tars plugin.

Reply:
That is sensitive about how we are going to write doc to be clear about this.

Reply:
I remove tars from the md document

Reply:
Oh, I didn't mean change log, you could make it back.

I mean here https://github.com/apache/skywalking-java/blob/main/docs/en/setup/service-agent/java-agent/Supported-list.md. Add supported protocols(including tars) in here clearly.

Reply:
Understood, thanks